scripts/ci/update-drone-config.py

```diff
@@ -0,0 +1,132 @@
+import os
+from hashlib import sha256
+from pathlib import Path
+from typing import Dict
+
+
+from botocore.config import Config
+from ruamel.yaml import YAML
+import argparse
+
+import boto3
+
+
+DRONE_FILE: str = ".drone"
+DRONE_FILE_SUFFIX: str = ".yml"
+REPOSITORY: str = "heliaxdev/juvix"
+
+STEP_NAME = 'script-integrity-check'
+
+files_to_check = [
+    'Makefile',
+    'scripts/push-changes.sh',
+    'scripts/format-and-org-gen.sh',
+    'scripts/check-formatting.sh',
+    'scripts/check-org-gen.sh',
+    'scripts/pre-run.sh'
+]
+
+scripts_to_run = [
+    'scripts/pre-run.sh'
```

Comment:
Do you want to add other scripts to run here?

Reply:
Maybe in the future